### ISSUE TITLE
help w/ setting custom cookie

### ISSUE THREAD

**Issue by davidbirdsong** [pingidentity/lua-resty-openidc#5]
hi there, great library. i'm wondering if you could give me pointer to where i could modify the set cookie action. we've been using tornado to do this in the past, but this library + openresty would be much better and would align w/ more of the rest of our stack.

we have a few pages that house a dashboard on admin.prodz.imgix.com and the page makes ajax calls to an api at api.prodz.imgix.com. w/ tornado, we did a set cookie on prodz.imgix.com once google sent the user back to the redirect url. i read through the code, but it didn't jump out at me where the right edits would yield this behavior. mind giving me a push in the right direction?

**Comment by zandbelt** [comment]
`lua-resty-openidc` depends on `lua-resty-session` for the session management once the authentication is done; you can control the domain for its cookie with `set $session_cookie_domain mydomain.org;`, as documented here: https://github.com/bungle/lua-resty-session#string-sessioncookiedomain (note there are various ways to pass that option)

let me know if that is what you're looking for

**Comment by davidbirdsong** [comment]
Thanks for the pointer. I think I'd tried this in the past, but I was running into another issue and falsely attributed to incorrect cookies.

Works great!
